R/gh_whoami.R

```diff
@@ -0,0 +1,74 @@
+#' Info on current GitHub user and token
+#'
+#' Reports wallet name, GitHub login, and GitHub URL for the current
+#' authenticated user, the first few and last characters of the token, and the
+#' associated scopes.
```

Comment:
How about we only show the first two letters now? Is that still useful for the user?

I'll also ask my security consultant, @jeroenooms. :)